### ISSUE TITLE
verification task requires Gradle 4.4+

### ISSUE THREAD

**Issue by jmcampanini** [nebula-plugins/nebula-publishing-plugin#97]
i believe the verification task requires gradle 4.4 or higher.

the current workaround i'm using is similar to the following:

```java
public static void disableNebulaVerificationBefore44(Project project) {
        GradleVersion minVersion = GradleVersion.version("4.4");
        if (GradleVersion.current().compareTo(minVersion) < 0) {
            project.getTasks().getByName("verifyPublication").setEnabled(false);
        }
    }
```

**Comment by chali** [comment]
Hi @jmcampanini, thank you for catching this issue. It will be solved by #98

**Comment by chali** [comment]
It is done and released in 7.0.4

**Comment by jmcampanini** [comment]
ty! :)
